src/graphql/account.rs

```diff
@@ -651,13 +643,13 @@ impl From<types::Account> for Account {
     }
 }
 
-fn strings_to_ip_addrs(ip_addrs: &[String]) -> Result<Vec<IpAddr>, AddrParseError> {
+fn strings_to_ip_addrs(ip_addrs: &[IpAddress]) -> Vec<IpAddr> {
     let mut ip_addrs = ip_addrs
         .iter()
-        .map(|ip_addr| ip_addr.parse::<IpAddr>())
-        .collect::<Result<Vec<_>, _>>()?;
+        .map(|ip_addr| ip_addr.0)
+        .collect::<Vec<IpAddr>>();
     ip_addrs.sort();
-    Ok(ip_addrs)
+    ip_addrs
```

Comment:
I think it would be better to remove duplicate values, although it is not directly related to this issue.

Reply:
I have added `ip_addrs.dedup();` to remove duplicates.